src/frontend/context/KorrigerVedtak/KorrigerVedtakSkjemaContext.ts

```diff
@@ -57,35 +53,23 @@ export const useKorrigerVedtakSkjemaContext = ({
     });
 
     useEffect(() => {
-        nullstillSkjema();
-    }, [korrigertVedtak?.vedtaksdato, korrigertVedtak?.begrunnelse]);
-
-    const valideringsstatuser = [
-        skjema.felter.vedtaksdato.valideringsstatus,
-        skjema.felter.begrunnelse.valideringsstatus,
-    ];
-
-    useEffect(() => {
-        if (
-            valideringsstatuser.some(
-                valideringsstatus => valideringsstatus === Valideringsstatus.IKKE_VALIDERT
-            )
-        ) {
-            validerAlleSynligeFelter();
-        }
-    }, [valideringsstatuser]);
+        const opprinneligVedtaksdato = korrigertVedtak
+            ? new Date(korrigertVedtak.vedtaksdato)
+            : undefined;
+        skjema.felter.vedtaksdato.validerOgSettFelt(opprinneligVedtaksdato);
+    }, []);
```

Comment:
Må heller sette skjemafeltet sin verdi til å være opprinnelig vedtaksdato her enn inne i opprettelsen av skjemafeltet for at `felt.nullstill()` skal funke i datovelgeren